guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>The <code>TimeSeriesNetwork</code> class can be used to make predictions of a feature layer or dataframe, or to predict a raster. </p><p>When making predictions of a feature layer or a dataframe, the dependent variable is assumed to be present in the input feature or raster, with the <code>seq_len</code> number of values already present. &nbsp;The <code>number_of_predictions</code> parameter controls the number of predictions to append to the existing sequence, however, if not specified, the model makes predictions equal to the number of null rows in the input feature. The index field is also populated if it is of type timestamp. </p><p>When making predictions of a raster, the function accepts only rasters. The values of all rasters for a given pixel are used to predict the value of that pixel in the output raster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>